iProdyHomework/src/homework14/AttandanceLogger.java

```diff
@@ -0,0 +1,87 @@
+package homework14;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.TreeMap;
+
+public class AttandanceLogger {
+    private String timestamp;
```

Comment:
Так будет намного элегантней. 
```suggestion
    private LocalTime timestamp;
```